website/site/content/docs/add-to-your-site.md

```diff
@@ -8,7 +8,7 @@ menu:
 
 # Add Netlify CMS to Your Site
 
-Netlify CMS is adaptable to a wide variety of projects. The only inflexible requirement is that your site content must be written in markdown, JSON, YAML, or TOML files, stored in a repo on [GitHub](https://github.com/). (If you're partial to another Git hosting service, check out the PRs in progress for [GitLab](https://github.com/netlify/netlify-cms/pull/517) and [Bitbucket](https://github.com/netlify/netlify-cms/pull/525) support.)
+Netlify CMS is adaptable to a wide variety of projects. The only inflexible requirement is that your site content must be written in markdown, JSON, YAML, or TOML files, stored in a repo on [GitHub](https://github.com/) or [GitLab](https://about.gitlab.com/). (Bitbucket support [coming soon](https://github.com/netlify/netlify-cms/pull/525).)
```

Comment:
Maybe we should also note that it is possible to use/create a custom backend.

Reply:
Oh, good point!

Reply:
Of course, it sure would be nice if we had a doc we could link to about how to make a custom backend... 😉

Reply:
You know what, I think you would be the perfect person to write that! 😉